packages/quantic/force-app/main/default/classes/SampleTokenProvider.cls

```diff
@@ -0,0 +1,9 @@
+global with sharing class SampleTokenProvider implements ITokenProvider {
+    public static String getHeadlessConfiguration() {
+        Map<String, String> headlessConfiguration = new Map<String,String>();
+        headlessConfiguration.put('organizationId', 'searchuisamples');
+        headlessConfiguration.put('accessToken','xx564559b1-0045-48e1-953c-3addd1ee4457');
```

Comment:
Is this a valid token for searchuisamples?

Reply:
Yes, it is. It's the one being used by the SearchUI etc